### PR TITLE
Vdm 69

### PR DESCRIPTION
**Pull request by deerwalk**

### CODE REVIEW

File: lib/python/vdm/server/HTTPListener.py [hunk 1]
```diff
@@ -2233,7 +2246,7 @@ def get(database_id):
                 try:
                     response = requests.get(url)
                 except Exception, err:
-                    return jsonify({'status': 'error', 'errorDetails': err, 'hostname': server[0]['hostname']})
+                    return jsonify({'status': 'error', 'hostname': server[0]['hostname']})
```
Comment (manjujames) [inline lib/python/vdm/server/HTTPListener.py:2249]: Why this change? 

Rest looks fine for 6.1.

Comment (deerwalk) [inline lib/python/vdm/server/HTTPListener.py:2249]: This change fixes an issue with starting clusters when any node is not available. It now displays proper error message when any node is not reachable

Comment (manjujames) [inline lib/python/vdm/server/HTTPListener.py:2249]: But, this looks like it is reverting the change.. May be this branch just needs to be sync'ed with master.
